assets/src/dashboard/app/font/fontProvider.js

```diff
@@ -49,9 +49,9 @@ function FontProvider({ children }) {
 
   const getFonts = useCallback(
     () =>
-      import(/* webpackChunkName: "chunk-fonts" */ '../../../fonts/fonts').then(
-        (res) => res.default
-      ),
+      import(
+        /* webpackChunkName: "chunk-fonts" */ '../../../../../packages/fonts/src/fonts.json'
```

Comment:
Is this the best way to import that? Can't it not be referenced by the package? So `web-stories-wp/...`?

Reply:
Good point!

Since the dashboard app is not yet a package itself, it might not be possible.

I'll have to test.

Reply:
Looks like it works 🎉